tapdb/sqlc/queries/assets.sql

```diff
@@ -321,7 +321,18 @@ JOIN genesis_info_view
 -- around that needs to be used with this query until a sqlc bug is fixed.
 LEFT JOIN key_group_info_view
     ON assets.genesis_id = key_group_info_view.gen_asset_id
-WHERE spent = FALSE
+LEFT JOIN managed_utxos utxos
```

Comment:
Correct, this was done on purpose. The initial approach was having a normal join with the `leased CASE` in the join clause, but this failed the `TestCommitBatchChainActions` unit test here: 
https://github.com/lightninglabs/taproot-assets/blob/c7fe6ef4afc9848cde8e19ee8e11f795fcaa0003/tapdb/asset_minting_test.go#L1044

The reason it failed is because that unit test expects to see the group genesis asset. But group genesis assets have `anchor_utxo_id` set to `NULL`, so they don't show up in the join unless it is a left join. Having the left join meant that I had to change the semantics of the query ever so slightly.

I now changed the join to have the semantics be consistent with `QueryAssets`while still making sure that the group genesis assets show up.